CHANGELOG.md

```diff
@@ -26,18 +26,21 @@ and includes an additional section for migration notes.
 - *ORCA-403* Added lambda post_to_queue_and_trigger_step_function to trigger step function for internal reconciliation.
 - *ORCA-309* Added lambda internal_reconcile_report_phantom to report entries present in the catalog, but not s3.
 - *ORCA-382* Added lambda internal_reconcile_report_orphan to report entries present in S3 bucket, but not in the ORCA catalog.
+- *ORCA-291* request_files lambda now accept "orcaRestoreTypeOverride" to override the glacier restore type at the workflow level. If desired, add "orcaRestoreTypeOverride": "{$.meta.collection.meta.restoreType}" to the workflow's task's task_config.
 
 ### Changed
 - *ORCA-299* `db_deploy` task has been updated to deploy ORCA internal reconciliation tables and objects.
 - *ORCA-161* Changed staged recovery SQS queue type from FIFO to standard queue.
 - SQS Queue names adjusted to include Orca. For example: `"${var.prefix}-orca-status-update-queue.fifo"`. Queues will be automatically recreated by Terraform.
 - *ORCA-334* Created IAM role for the extract_filepaths_for_granule lambda function, attached the role to the function
 - *ORCA-404* Updated shared_db and relevant lambdas to use secrets manager ARN instead of magic strings.
+- *ORCA-291* Updated request_files lambda and terraform so that the glacier restore type can now be overwritten by the user instead of being hard-coded previously during deployment.
```

Comment:
Perhaps should have a separate sentence explaining that the default is also now configurable in tfvars.

Reply:
yes what I meant was that the user can now set the default value for restore type during deployment in terraform. I guess I will update the wording

Reply:
updated to 

`Updated request_files lambda and terraform so that the glacier restore type can be set via terraform during deployment or overridden via a change in the collections configuration.`

Reply:
Close. I don't know how people use this well enough to know if `"orcaDefaultRecoveryTypeOverride": "{$.meta.collection.meta.orcaDefaultRecoveryTypeOverride}"` will be helpful. `orcaDefaultRecoveryTypeOverride` definitely feels like an internal detail. @bhazuka?